site/app/libraries/Core.php

```diff
@@ -139,6 +139,13 @@ public function loadConfig($semester, $course) {
         }
     }
 
+    public function loadMasterConfig() {
+        $conf_path = FileUtils::joinPaths(__DIR__, '..', '..', '..', 'config');
+
+        $this->config = new Config($this, '', '');
```

Comment:
Remove the second and third parameters off the class constructor and make it just accept `Core $core`.